modules/distr-tracing-tempo-config-multitenancy.adoc

```diff
@@ -159,9 +159,21 @@ spec:
             authenticator: bearertokenauth
           headers:
             X-Scope-OrgID: "dev"
+        otlphttp/dev: # <2>
+          endpoint: https://tempo-simplest-gateway.chainsaw-multitenancy.svc.cluster.local:8080/api/traces/v1/dev
+          tls:
+            insecure: false
+            ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+          auth:
+            authenticator: bearertokenauth
+          headers:
+            X-Scope-OrgID: "dev"
       service:
         extensions: [bearertokenauth]
         pipelines:
           traces:
-            exporters: [otlp/dev]
+            exporters: [otlp/dev] # <3>
 ----
+<1> OTLP gRPC Exporter.
+<2> OTLP HTTP Exporter.
```

Comment:
I'm not seeing "OTLP" expanded in the document. Maybe it should be on first use?

Reply:
It's part of the exporter name.